code/modules/mob/living/carbon/carbon.dm

```diff
@@ -106,6 +106,8 @@
 			Weaken(3)
 	if(override)
 		return override
+	else if(shock_damage > 400)
+		gib()

```

Comment:
can you make this ```dust()```? electrocution should turn you into dust to go with the cartoon themes I gave electricity (like shock skeletons) (also, if dust doesn't drop a brain, give it an arg to do that, because leaving no brain is a bit cruel)